Planovac_Udalosti/Program.cs

```diff
@@ -35,15 +33,9 @@ static void Main(string[] args)
 
                         try
                         {
-                            string[] dateOfEventSplit = dateOfEvent.Split("-", StringSplitOptions.RemoveEmptyEntries);
-
-                            int yearOfEvent = int.Parse(dateOfEventSplit[0]);
-                            int monthOfEvent = int.Parse(dateOfEventSplit[1]);
-                            int dayOfEvent = int.Parse(dateOfEventSplit[2]);
-
-                            DateTime parsedDate = new DateTime(yearOfEvent, monthOfEvent, dayOfEvent);
+                            DateTime parsedDate = DateTime.ParseExact(dateOfEvent, "yyyy-MM-dd", null);
```

Comment:
Ve standardni knihovne uz mame pripravene metody na nacitani `DateTime`, takze si kod muzeme hodne zjednodusit. Kazdopadne i bez pomocne metody se ti to povedlo, vymyslela jsi pro to vlastni implementaci 👏